core/dbt/task/list.py

```diff
@@ -56,6 +58,7 @@ def __init__(self, args, config):
     def pre_init_hook(cls, args):
         """A hook called before the task is initialized."""
         log_manager.stderr_console()
+        event_logger.STDOUT_LOG.level = logging.WARN
```

Comment:
I'm definitely not a fan of reaching in and modifying `STDOUT_LOG` like this, but I'm fine with it as a quick fix. Can we just leave a comment here stating that this is a hack and describe what the right way is?

Reply:
actually, are there info-level logs we're trying to filter out? I'm not sure why we're setting it to warn here.